longest.py

```diff
@@ -0,0 +1,6 @@
+"""longest."""
+
+
+def longest(s1, s2):
+    """Return the sorted string of unique elements in s1 or s2."""
+    return ''.join(sorted(list(set(s1).union(set(s2)))))
```

Comment:
No need to cast your set to a list here; any iterable (of which there are very many kinds!), including a set, can be passed into sorted.